casper/contracts/simple_casper.v.py

```diff
@@ -1,3 +1,12 @@
+#  List of events the contract logs
+# Withdrawal address used always in _from and _to as it's unique and validator index is removed after some events
+Deposit: __log__({_from: indexed(address), _validator_index: indexed(num), _validation_address: address, _start_dyn: num, _amount: num(wei)})
+Vote: __log__({_from: indexed(address), _validator_index: indexed(num), _target_hash: bytes32, _target_epoch: num, _source_epoch: num})
```

Comment:
Same for `Vote`... should probably index `target_epoch`. We get three per event and probably don't want to later look back and say "shit, i wish we indexed that."